pkgs/applications/misc/foot/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, fcft, freetype, pixman, libxkbcommon, fontconfig, wayland
+, meson, ninja, ncurses, scdoc, tllist, wayland-protocols, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "foot";
+  version = "1.4.4";
+
+  src = fetchgit {
+    url = "https://codeberg.org/dnkl/foot.git";
```

Comment:
nixpkgs-review keeps failing to me because the server is unreachable...

Reply:
But I _can_ access it via the web.

Reply:
weird… nixpkgs-review works fine for me. maybe try cloning the repo manually somewhere maybe will yield a helpful error message?!